index.html

```diff
@@ -13,34 +13,135 @@
     <body>
         <div class="grid-container">
             <nav class="navbar flex--row flex--justify-between flex--align-center">
-                <a href="/" class="brand flex--row flex--align-center">
+                <a href="/" class="brand flex--row flex--align-center flex--justify-center">
                     <img 
-                        src="./logo-assets/elixir-logo.png" 
+                        src="../logo-assets/elixir-logo.png" 
```

Comment:
Just a suggestion. You can use the GitHub raw link for images so even if you want to use it somewhere else you wouldn't have to keep the logo-asset folder in that project